modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -109,24 +135,54 @@ exports.changeProfilePicture = function (req, res) {
   function deleteOldImage() {
     return new Promise(function (resolve, reject) {
       if (existingImageUrl !== User.schema.path('profileImageURL').defaultValue) {
-        fs.unlink(existingImageUrl, function (unlinkError) {
-          if (unlinkError) {
+        if (config.uploads.storage === 's3' && config.aws.s3) {
```

Comment:
Since this check is being used in multiple places, you could move it into a local variable at the top level of this request handler.

Reply:
Maybe something like: `var useLocalStorage = config.uploads.storage === 'local';`

Then here you can just ` if (!useLocalStorage) `

Reply:
I wanted to let place for another storage provider... 
But if it will only be local or S3, I'll do it.